<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpeningDialog</class>
 <widget class="QDialog" name="OpeningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sumuri Hex Viewer V1</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="outputDirectoryPushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>EULA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="caseNolineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="caseNamelineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>55</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>140</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Source</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>129</y>
     <width>641</width>
     <height>43</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>43</height>
    </size>
   </property>
   <widget class="QPushButton" name="selectSourceButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectSourcelineEdit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>0</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>277</width>
      <height>0</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>Select Forensic Image / Physical Drive ...</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>55</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Case Name</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>89</y>
     <width>358</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <widget class="QRadioButton" name="radioButtonPhysicalDrive">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>12</y>
      <width>163</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Physical Drive</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonForensicImage">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>164</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Forensic image</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>180</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Directory</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>179</y>
     <width>591</width>
     <height>39</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>39</height>
    </size>
   </property>
   <widget class="QLineEdit" name="outputDirectorylineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>249</width>
      <height>0</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>Select Directory for Case...</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>99</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Case No. *</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>222</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelLoading">
      <property name="minimumSize">
       <size>
        <width>133</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadCasepushButton">
      <property name="text">
       <string>Load Case</string>
      </property>
      <property name="icon">
       <iconset theme="document-open"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpeningDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpeningDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
